### ISSUE TITLE
Comparing OSM data with government data

### ISSUE THREAD

**Issue by Dimitar5555** [osm-fr/osmose-backend#2182]
The Bulgarian road agency has released a dataset of the republican and municipal road routes in Bulgaria. They have `ref` and geometry which can be matched to OSM relations (with `network=bg:national` and `network=bg:municipal` for respectively republican and municipal roads). Can Osmose detect the differences and report them?

The data is hosted on their website (https://api.bg/bg/gis.html) inside `.shp` files.

**Comment by frodrigo** [comment]
Yes it is possible to do this. It as already done for France and Spain.
https://osmose.openstreetmap.fr/en/issues/open?item=7170&class=13,200
For this tow countries it is only done on geometrical matching.

As first step I also only recommend to match only geometry, and when few geometry as missing, also check for ref.

Something like this must be setup
https://github.com/osm-fr/osmose-backend/blob/dev/analysers/analyser_merge_road_ES.py
https://github.com/osm-fr/osmose-backend/blob/dev/analysers/analyser_merge_road_FR.py

(The doc https://github.com/osm-fr/osmose-backend/tree/dev/doc)

You can try to make a file like this. Even if you are not able to run it your self, it help to find the fields and the OSM tags required.
We can help to setup and test it.
